content/200-orm/500-reference/050-prisma-client-reference.mdx

```diff
@@ -1255,6 +1255,54 @@ const users = await prisma.user.createMany({
 })
 ```
 
+### `createManyAndReturn()`
+
+`createManyAndReturn` creates multiple records and returns the resulting objects.
+
+:::info
+
+`createManyAndReturn()` is only available in Prisma ORM version 5.14.0 and up.
+`createManyAndReturn()` is only available for PostgreSQL, CockroachDB, and SQLite.
+
+:::
+
+**Options**
```

Comment:
```suggestion
#### Options
```

Reply:
This suggested change will change the numbering of all the anchors for the "Options" headlines below this one and will require manual updates and fixes across the whole docs unfortunately.